client/src/lib/components/Cards/Card.tsx

```diff
@@ -111,7 +113,13 @@ export const Card: React.FC<CardProps> = ({
           <Header>
             {title && <Display20 numberOfLines={2}>{title}</Display20>}
             <Spacer4 />
-            <Byline pictureURL={hostPictureURL} name={hostName} />
+            {onHostPress ? (
```

Comment:
Another way of doing this is by defining an empty callback as default for the onHostPress, and maybe `disabled={!onHostPress}` if we wanna make it explicit I guess.

But the current approach is also legit, just pointing another possible way of doing it 😁

Reply:
Ah, right, the disabled would mitigate that I guess